addons/event/migrations/9.0.0.1/post-migration.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2016 Tecnativa - Vicent Cubells <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def update_seats_availability(cr):
+    cr.execute("""
+                UPDATE event_event
```

Comment:
You don't need to indent so much these lines. One indent level more than previous line is enough.